docs/app/pages/components/components-sections/dashboard/dashboard/dashboard.component.html

```diff
@@ -215,6 +215,7 @@ <h3 class="widget-title">Alert</h3>
     <tr uxd-api-property name="minRowSpan" type="number" defaultValue="1">Defines the minimum number of rows this widget should occupy.</tr>
     <tr uxd-api-property name="resizable" type="boolean">Defines whether or not this widget can be resized.</tr>
     <tr uxd-api-property name="autoPositioning" type="boolean" defaultValue="true">Defines whether or not this widget will be automatically repositioned to fill any vacant space above it.</tr>
+    <tr uxd-api-property name="pinned" type="boolean" defaultValue="false">Defines whether or not this widget is pinned to the top row and whether it can be moved from its location. When set to <code>true</code> this widget will need a <code>row</code> and <code>col</code> specified.</tr>
```

Comment:
As usual, please break long lines. I would not expect `row` and `col` to have to be specified for this to work. `col` should implicitly be 0 if it is pinned.